cms/djangoapps/contentstore/tests/test_course_settings.py

```diff
@@ -1290,10 +1289,9 @@ def test_post_settings_with_staff_not_enrolled(self, mock_request):
         })
         self.assertEqual(response.status_code, 200)
 
-    @override_waffle_flag(ENABLE_PROCTORING_PROVIDER_OVERRIDES, True)
-    def test_proctoring_provider_present_when_waffle_flag_enabled(self):
+    def test_proctoring_provider_present(self):
```

Comment:
Since we're removing this flag, and the proctoring provider field will always be visible, do you think we still need this test (and the other proctoring provider tests in this file)? It looks like many of these tests are for settings that are feature flagged in some way.

Reply:
I think these tests could also be removed:

- https://github.com/edx/edx-platform/pull/24606/files#diff-6fa4b8c19497e771bdaaed83c5568695R1346
- https://github.com/edx/edx-platform/pull/24606/files#diff-6fa4b8c19497e771bdaaed83c5568695R1367
- https://github.com/edx/edx-platform/pull/24606/files#diff-6fa4b8c19497e771bdaaed83c5568695R1294

Let me know if you think there are others, but the other tests are for ensuring that an en escalation email is required if proctortrack is selected, and that course staff cannot modify the proctoring provider after the course start date.